### DISCUSSION TITLE
DNS and SSL Certificate

### DISCUSSION THREAD

**Discussion by Zhengda-Huang** [vercel/community#5348] [Help] [answered]
### Summary

I recently purchase a domain name from square space. Currently I have it connect my vercel project to square space domain by using nameserver. I used the name server given when I input into the domain name in the dashboard. However, I'm having trouble to generate SSL certificate on the square space side resulting in trouble for reaching the website. I have contacted the square customer support but they ask me to ask vercel support for help. Any help is appreciate thank you

Furthermore, When was able to connect the website to the domain name. After about 20 mins when I go search the website it prompt 
for my website. Does anyone have any idea on the issue. Thank you
This site can’t be reached
Check if there is a typo in www.uniquenailspas.com.
DNS_PROBE_FINISHED_NXDOMAIN

### Steps to Reproduce

1. Purchase Domain name from square space
2. connect vercel project with square space using name server 
3. try to generate SSL certificate

**Comment by swarnava** [comment, marked as answer]
HI there,

It sounds like DNS propagation issue which is updated now. As I can see your website is accessible now, are you still facing this issue?

**Reply by Zhengda-Huang** [reply]
The problem seem to be fixed. Thank you
